### ISSUE TITLE
Idea: Nicer assertions for collections

### ISSUE THREAD

**Issue by rtfeldman** [elm-community/elm-test#31]
These would be a nice way to give better output for collections, e.g. incorporating diffs, expected length vs actual length, etc.

Some ideas:

```elm
Expect.equalLists : List a -> List a -> Expectation
```

```elm
Expect.equalSets : Set a -> Set a -> Expectation
```

```elm
Expect.equalDicts : Dict a -> Dict a -> Expectation
```

**Comment by lpil** [comment]
I think I prefer these names
```elm
Assert.equalLists
Assert.equalSets
Assert.equalDicts
```

**Comment by rtfeldman** [comment]
> Also is `Assert` the same as `Expect`?

Whoops! `elm-test` 1.0 used `Assert` and I forgot to update it when I copy/pasted this issue over. 😅

**Comment by rtfeldman** [comment]
Also, I liked your names better so I updated the description to use them!

**Comment by mgold** [comment]
So these would have the same semantics as `Expect.equal`, but they'd just provide more information in the failing case?

**Comment by rtfeldman** [comment]
@mgold yeah, exactly!

**Comment by rtfeldman** [comment]
Fixed in #46
